include/vsb.h

```diff
@@ -123,6 +123,11 @@ void		 VSB_destroy(struct vsb **);
 	 * Not valid with VSB_QUOTE_JSON and VSB_QUOTE_HEX
 	 */
 
+#define VSB_QUOTE_CLI		64
+	/*
+	 * Add quotes ".." around arguments when needed
+	 */
```

Comment:
I think the rule for "when needed" is the presence of either space, tabulation, or double quote in the argument value, so in its current state `VSB_QUOTE_CLI` is insufficient.